tasks/downloadArchiwalnyBdotTask.py

```diff
@@ -26,16 +27,13 @@ def run(self):
                 return False
             else:
                 self.page_exist = 'YES'
-                QgsMessageLog.logMessage('pobieram ' + self.url)
+                QgsMessageLog.logMessage(f'pobieram {self.url}')
                 # fileName = self.url.split("/")[-1]
-                service_api.retreiveFile(url=self.url, destFolder=self.folder, obj=self)
-                if self.isCanceled():
-                    return False
-                return True
+                self.result, self.exception = service_api.retreiveFile(url=self.url, destFolder=self.folder, obj=self)
+                return not self.isCanceled()
```

Comment:
Trochę chyba to nie działa, bo zawsze mi zwraca True

Reply:
szczerze to nie zagłębiałam się w poprawność działania - ja tylko zrefaktorowałam ten fragment żeby zamiast 3 linii była 1

Reply:
A to faktycznie, teraz lepiej wyglada :) ale jednak dobrze, żeby jeszcze odnajdywał odpowiedź poprawnie